redshift_admin.view.lkml

```diff
@@ -0,0 +1,337 @@
+    type: string
+    sql: ${TABLE}."table" ;;
+  }
+
+  dimension: encoded {
+    type: yesno
+    sql: case ${TABLE}.encoded
+        when 'Y'
+        then true
+        when 'N'
+        then false
+        else null
+      end
+       ;;
+  }
+
+  dimension: distribution_style {
+    type: string
+    sql: ${TABLE}.diststyle ;;
+  }
+
+  dimension: sortkey {
+    type: string
+    sql: ${TABLE}.sortkey1 ;;
+  }
+
+  dimension: max_varchar {
+    type: number
+    sql: ${TABLE}.max_varchar ;;
+  }
+
+  dimension: sortkey_encoding {
+    type: string
+    sql: ${TABLE}.sortkey1_enc ;;
+  }
+
+  dimension: number_of_sortkeys {
+    type: number
+    sql: ${TABLE}.sortkey_num ;;
+  }
+
+  dimension: size {
+    type: number
+    sql: ${TABLE}.size ;;
+  }
+
+  dimension: pct_used {
+    type: number
+    sql: ${TABLE}.pct_used ;;
+  }
+
+  dimension: empty {
+    type: number
+    sql: ${TABLE}.empty ;;
+  }
+
+  dimension: unsorted {
+    type: number
+    sql: ${TABLE}.unsorted ;;
+  }
+
+  dimension: stats_off {
+    type: number
+    sql: ${TABLE}.stats_off ;;
+  }
+
+  dimension: rows_in_table {
+    type: number
+    sql: ${TABLE}.tbl_rows ;;
+  }
+
+  dimension: skew_sortkey {
+    type: number
+    sql: ${TABLE}.skew_sortkey1 ;;
+  }
+
+  dimension: skew_rows {
+    type: number
+    sql: ${TABLE}.skew_rows ;;
+    html: {% if value < 25 %}
+      <div style="color:#B40404; background-color:#22CE7E; font-size:100%; text-align:center">{{ rendered_value }}</div>
+      {% elsif value >= 25 and value < 50 %}
+      <div style="color:#868A08; background-color:#95F047; font-size:100%; text-align:center">{{ rendered_value }}</div>
+      {% elsif value >= 50 %}
+      <div style="color:#868A08; background-color:#C64646; font-size:100%; text-align:center">{{ rendered_value }}</div>
+      {% endif %}
+      ;;
+  }
+}
+
+view: db_space {
+  derived_table: {
+    sql: select name as table
+        , trim(pgn.nspname) as schema
+        , sum(b.mbytes) as megabytes
+        , sum(a.rows) as rows
+      from (select db_id
+              , id
+              , name
+              , sum(rows) as rows
+            from stv_tbl_perm a
+            group by 1,2,3) as a
+      join pg_class as pgc
+      on pgc.oid = a.id
+      join pg_namespace as pgn
+      on pgn.oid = pgc.relnamespace
+      join pg_database as pgdb
+      on pgdb.oid = a.db_id
+      join (select tbl
+              , count(*) as mbytes
+            from stv_blocklist
+            group by 1) as b
+      on a.id = b.tbl
+      group by 1,2
+       ;;
+  }
+
+  # dimensions #
+
+  dimension: table {
+    type: string
+    sql: ${TABLE}.table ;;
+  }
+
+  dimension: schema {
+    type: string
+    sql: ${TABLE}.schema ;;
+  }
+
+  dimension: megabytes {
+    type: number
+    sql: ${TABLE}.megabytes ;;
+  }
+
+  dimension: rows {
+    type: number
+    sql: ${TABLE}.rows ;;
+  }
+
+  dimension: table_stem {
+    sql: case
+        when (${table} ~ '(lr|lc)\\$[a-zA-Z0-9]+_.*')
```

Comment:
Does Looker still use the `lc` prefix? I feel like this may have changed and I never updated this code.